runtimes/eden/src/pallets_util.rs

```diff
@@ -163,12 +163,18 @@ impl Default for SponsorshipType {
 	}
 }
 
+parameter_types! {
+	pub const PotDeposit: Balance = 1000 * constants::NODL;
```

Comment:
Probably a little cheap considering existing NODL price.

Reply:
It's just an example, can you tell me what would be a better price?

Reply:
Based on our conversation today, let's keep this low so we do not limit any possible cool usage. We can  reevaluate later